MobileWallet/Screens/RestoreWallet/RestoreWalletPendingView.swift

```diff
@@ -0,0 +1,154 @@
+//  RestoreWalletPendingView.swift
```

Comment:
It feels like this one should be a separate view controller that carries out the restoration. Let's talk about this too.

Reply:
Its just  stub while backup is pending. All backup process presents into iCloudBackup class. I think we should not do separated controller for that